src/mantine-core/src/components/Pagination/Pagination.story.tsx

```diff
@@ -16,4 +16,11 @@ storiesOf('@mantine/core/Pagination', module)
   .add('With 2 boundaries', () => <Pagination total={20} boundaries={2} initialPage={12} />)
   .add('With 2 boundaries and 2 siblings', () => (
     <Pagination siblings={2} boundaries={2} total={20} initialPage={12} />
+  ))
+  .add('With jump to first/last buttons', () => (
+      <div style={{ padding: 40 }}>
+        <Pagination total={20} boundaries={2} initialPage={12} showFirstButton />
```

Comment:
Let's combine showFirstButton and showLastButton props into one and name it `withEdges`

Reply:
I thought the above approach allows a more fine grained control over whether a user wants to render single or both the icons ? With `withEdges` both get rendered at once, right ?

Reply:
Yes, it would be easier like this